### ISSUE TITLE
how to install ethminer on ubuntu 16.10

### ISSUE THREAD

**Issue by wthhelp** [ethereum/cpp-ethereum#4027]
I am unable to install ethminer on ubuntu 16.10
I used the following commands

sudo apt-get clean 
sudo add-apt-repository -y ppa:ethereum/ethereum-qt 
sudo add-apt-repository -y ppa:ethereum/ethereum 
sudo apt-get update 
sudo apt-get install cpp-ethereum

when trying to install i get the error:

sudo apt-get install cpp-ethereum
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package cpp-ethereum

how do i install it?

**Comment by chfast** [comment]
You are following outdated instructions. Is this helpful? https://github.com/ethereum/cpp-ethereum#mining

**Comment by wthhelp** [comment]
Hello and thank you for your reply,

Unfortunately I already have switched over to windows 8 for not knowing how to use linux based OS properly.

Currently synchronising, I think I will succeed :)

Hanki Outlook for Android<http://aka.ms/ghei36>



On Mon, Apr 3, 2017 at 7:22 PM +0300, "Paweł Bylica" <notifications@github.com<mailto:notifications@github.com>> wrote:


You are following outdated instructions. Is this helpful? https://github.com/ethereum/cpp-ethereum#mining

**Comment by wthhelp** [comment]
Hello again,

I am still having problems with ethereum.
I synchronized the blockchain with my main pc (it has a faster processor) and now I want to move the blockchain to my mining rig. I tried exporting and importing but the import seems to take forever. I also tried copy/paste directly to the appdata folder but that did not help either, the geth sync starts from the beginning anyway. Also I am unable to create a wallet in ethereum. It keeps telling me I dont have an account yet.

Any help would be appreciated

Hanki Outlook for Android<http://aka.ms/ghei36>



On Mon, Apr 3, 2017 at 7:22 PM +0300, "Paweł Bylica" <notifications@github.com<mailto:notifications@github.com>> wrote:


You are following outdated instructions. Is this helpful? https://github.com/ethereum/cpp-ethereum#mining
